OoyalaSkinSampleApp/app/src/main/java/com/ooyala/sample/players/IntentBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.ooyala.sample.players;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.Intent;
+
+public class IntentBuilder {
+
+  private Class<? extends Activity> activityClass;
+  private Context context;
```

Comment:
I think it's not good to store Activity context as field, better to pass it in methods where it is needed.

Reply:
But you can create Builder object as field, Singleton and so on so I think you must create safe builder at the beginning

Reply:
? 
Is it so hard to pass context in build() method?

Reply:
ok, but now this is not a classical builder pattern

Reply:
I know that, but that's different kind of things =)